python/kvikio/kvikio/_lib/remote_handle.pyx

```diff
@@ -67,6 +76,59 @@ cdef class RemoteFile:
         ret._handle = make_unique[cpp_RemoteHandle](move(ep), n)
         return ret
 
+    @classmethod
+    def open_s3(
+        cls,
+        bucket_name: str,
+        object_name: str,
+        nbytes: Optional[int],
+    ):
+        cdef RemoteFile ret = RemoteFile()
```

Comment:
*nit* throughout (and in the existing usages);
```suggestion
        cdef RemoteFile ret = cls()
```

Or else you should use a `staticmethod`.

Reply:
fixed